Task2.py

```diff
@@ -0,0 +1,19 @@
+# Multi-paradigm programming languages, Task №2
+# Andrey Fortus IKM-221a
+
+GENERAL_INFO = 'Multi-paradigm programming languages, Task №2'
+STUDENT_INFO = 'Andrey Fortus IKM-221a'
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+# Math
+x = float(input('Enter x value: '))
```

Comment:
what if a user is going to input such str "Andrey Fortus". Ho the program deals with such kind of input.